KinSDK/KinSDK/KinTypes.swift

```diff
@@ -126,6 +96,7 @@ public struct PaymentInfo {
      Amount in `Kin` of the payment
      */
     public var amount: Kin {
+        // ???: the amount should be quark going into kin and not kin into kin
```

Comment:
@yosriz this is a comment for me to look into at a later time. it appears at first glance like its wrong.